Environment-Variables.md

```diff
@@ -48,6 +48,7 @@ node server/server.js --host=127.0.0.1 --port=8080
 | `UPTIME_KUMA_IN_CONTAINER`| | (1.23.0) Is Uptime Kuma inside a container? | | |
 
 ## MariaDB Environment Variables
+Only Version 2
 
```

Comment:
Could you please use the `(2.0.0) ` for the description formatting as above? ^^

Reply:
ok i update the file